eogrow/core/area/batch.py

```diff
@@ -91,8 +92,8 @@ def _create_grid(self) -> Dict[CRS, GeoDataFrame]:
         return grid
 
     def _verify_batch_request(self, batch_request: BatchRequest) -> None:
-        """Verifies that given batch request has finished and that it has the same tiling grid parameters as
-        they are written in the config.
+        """Verifies that the given batch request has finished and that it has the same tiling grid parameters as
```

Comment:
perhaps this makes the reference to the request object more prominent 

```suggestion
        """Verifies that the given batch request has finished and that it contains the same tiling grid parameters as
```